examples/coreos/units-etcd/ceph-rgw@.service

```diff
@@ -0,0 +1,24 @@
+[Unit]
+Description=Rados Gateway Service
+After=docker.service
+
+[Service]
+EnvironmentFile=/etc/environment
+Environment=RGW_NAME=%H
+ExecStartPre=-/usr/bin/docker kill %p
+ExecStartPre=-/usr/bin/docker rm %p
+ExecStart=/usr/bin/docker run --rm --name %p -p 80:80 \
+   --net=host \
```

Comment:
I'd prefer to not have `--net=host` as a default in an example, where it isn't needed.  rgw works just fine without it.